sensor_msgs/LaserScan laser
geometry_msgs/Vector3 relativ_goal_pos
